config/api/routes/files.php

```diff
@@ -7,7 +7,13 @@
  * Files Routes
  */
 return [
-
+	[
+		'pattern' => 'files/@/(:any)',
+		'method'  => 'GET',
+		'action'  => function (string $id) {
+			return $this->file('file://' . $id, false);
```

Comment:
Doesn't this call `Cms\Api::file()`? This method expects a string as second argument. I think it needs to be:

```suggestion
			return $this->file('file://' . $id);
```

And then `Api::file()` needs to support the `null` value.